_layouts/base.html

```diff
@@ -142,6 +142,9 @@
             <div class="grid-col-auto">
               <h3 class="usa-footer__logo-heading">U.S. General Services Administration</h3>
             </div>
+            <div class="grid-col-fill text-right">
```

Comment:
Should we allow this to collapse to a new line in smaller viewports? It gets a bit tight. We can use [USWDS responsive class prefixes](https://designsystem.digital.gov/utilities/layout-grid/#responsive-classes) to customize it per-viewport range.

![image](https://user-images.githubusercontent.com/1779930/103007729-eb4e6780-4501-11eb-914b-171fa765d19a.png)

Reply:
Great suggestion, fixed!